src/components/Input/Input.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { HTMLInputTypeAttribute } from "react";
+import "./input.css";
+
+export interface InputProps {
+  /**
+   * The id of the input element. Use this prop to make value and placeholder accessible for screen readers.
+   */
+  id?: string;
+
+  /**
+   * The HTML type of the input element
+   */
+  type?: HTMLInputTypeAttribute;
+
+  /**
+   *  It comes with two variants: outlined (default) and filled
+   */
+  variant?: "outlined" | "filled"
```

Comment:
Yang fillednya I think ada bug, Can help me to check bugnya soalnya hasilnya begini 
![image](https://user-images.githubusercontent.com/51783522/133741376-6faa96fd-353e-49f8-9f51-b873d16dc397.png)

i think lebih bagus kalo dia kaya outline, cuma ada background aja

Reply:
Hmm ini actually bkn bug lol itu deliberate. Gw copas dri [MUI](https://mui.com/components/text-fields/)

![image](https://user-images.githubusercontent.com/41280211/133782911-174221c7-6233-4f4d-ae72-2e3e633ce276.png)
mui.com/components/text-fields/).

Reply:
oh hem boleh juga sih